src/main/webapp/scripts/toggle.js

```diff
@@ -18,6 +20,12 @@ let loadingCounter;
  * Returns a promise that resolves when everything loads.
  */
 async function loadContentSection() {
+  let location = getCookie('location');
+  if (location != "") {
+    let lat = getCookie("lat");
+    let lng = getCookie("lng");
+    locationObj = await loadObject(`${LOCATION_OBJ_URL}?lat=${lat}&lng=${lng}`);
```

Comment:
I think we could make this call in the main page, and save lat, lng and address as a cookie array. That way we don't have to load the same object multiple times.

Reply:
I was thinking about that earlier -- I would think it would save time, but how do you store a cookie array? I thought that all cookies had to be String objects.